index.html

```diff
@@ -2570,39 +2589,27 @@
             <tr>
               <td><dfn><code id="idl-def-RTCDtlsRole.auto">auto</code></dfn></td>
               <td>
-                <p>The DTLS role is determined based on the resolved ICE role: the
-                <code>controlled</code> role acts as the DTLS client, the
-                <code>controlling</code> role acts as the DTLS server. Since
-                <code><a>RTCDtlsRole</a></code> is initialized to <code>auto</code> on
-                construction of an <code><a>RTCDtlsTransport</a></code> object,
-                <code>transport.getLocalParameters().RTCDtlsRole</code> will have an
-                initial value of <code>auto</code>.</p>
+                <p>The DLTS role is determined based on the resolved ICE role:
+                the ICE <code>controlled</code> role acts as the DTLS client and
+                the ICE <code>controlling</code> role acts as the DTLS server.
+                <code>getLocalParameters().role</code> always has a
+                value of <code>auto</code>.</p>
```

Comment:
Completely agreed. It makes no sense to have a getter that provides nothing useful.